pom.xml

```diff
@@ -226,6 +226,11 @@
                 <artifactId>org.wso2.carbon.humantask</artifactId>
                 <version>${carbon.business-process.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.wso2.carbon.identity.workflow.engine</groupId>
+                <artifactId>workflow.engine</artifactId>
+                <version>1.0-SNAPSHOT</version>
```

Comment:
Let's define the version as constant.
refer : ` <version>${carbon.business-process.version}</version>`